CMakeLists.txt

```diff
@@ -42,11 +41,12 @@ set(SYM_RUNTIME_BUILD_ARGS
   -DCMAKE_SHARED_LINKER_FLAGS_INIT=${CMAKE_SHARED_LINKER_FLAGS_INIT}
   -DCMAKE_MODULE_PATH=${CMAKE_MODULE_PATH}
   -DCMAKE_SYSROOT=${CMAKE_SYSROOT}
-  -DQSYM_BACKEND=${QSYM_BACKEND}
+  -DSYMCC_RT_BACKEND=${SYMCC_RT_BACKEND}
+  -DLLVM_VERSION=${LLVM_VERSION}
   -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
   -DZ3_TRUST_SYSTEM_VERSION=${Z3_TRUST_SYSTEM_VERSION})
 
-ExternalProject_Add(SymRuntime
+ExternalProject_Add(SymccRuntime
```

Comment:
Like above, should we change the spelling to `SymCCRuntime`?

Reply:
Fine for me :)